src/templates/HomePageTemplate.js

```diff
@@ -87,12 +87,12 @@ const HomePageTemplate = ({
               <h2>News</h2>
             </div>
             {news &&
-              news.map((article, index) => {
+              news.map(({ node }, index) => {
                 return (
-                  <div key={index} className="col-md-10">
-                    <p className="c-green">March 24, 2018</p>
-                    <p>{article}</p>
-                    <p>Read More</p>
+                  <div key={index} className="col-12">
+                    <p className="c-green">{node.frontmatter.date}</p>
+                    <ReactMarkdown source={node.frontmatter.text} />
```

Comment:
Vi borde ha typ en liten description på hemsidan istället för all text? Typ såhär:
https://github.com/netlify-templates/gatsby-starter-netlify-cms/blob/master/src/pages/blog/2017-01-04-just-in-small-batch-of-jamaican-blue-mountain-in-store-next-week.md